src/snackbar.jsx

```diff
@@ -162,7 +161,7 @@ const Snackbar = React.createClass({
 
       //Only Bind clickaway after transition finishes
       this.timerTransitionId = setTimeout(() => {
-        this._bindClickAway();
+        this.timerTransitionId = 0;
```

Comment:
To elaborate further, since this component no longer has a bindClickAway function, it binds by providing an onClickHandler to the ClickAwayListener component.